libs/SmartSync/SmartSync/Classes/Manager/SFSmartSyncSyncManager.m

```diff
@@ -515,22 +516,41 @@ - (void)resumeSyncUpOneEntry:(SFSyncState*)sync
     };
     
     // Create handler
-    SFSyncServerTargetCompleteBlock completeBlockCreate = ^(NSDictionary *d) {
+    SFSyncUpTargetCompleteBlock completeBlockCreate = ^(NSDictionary *d) {
         // Replace id with server id during create
         record[kId] = d[kSyncManagerLObjectId];
         completeBlockUpdate(d);
     };
     
-    SFSyncServerTargetCompleteBlock completeBlock;
+    
+    SFSyncUpTarget *target = (SFSyncUpTarget*) sync.target;
+    NSArray *fieldList = sync.options.fieldlist;
+    
+    // Getting type and id
+    NSString* objectType = [SFJsonUtils projectIntoJson:record path:kObjectTypeField];
+    NSString* objectId = record[kId];
+    
+    // Fields to save (in the case of create or update)
+    NSMutableDictionary* fields = [NSMutableDictionary dictionary];
```

Comment:
obj[""] will return nil, but I thought we were using [SFJsonUtils projectIntoJson:record path:fieldName]; -- which we are not.
If I switch it over to using "project", then it will allow people to get the whole record with "", or nested field values with paths like "a.b.c" and behave like before for regular attributes.